contracts/misc/UiPoolDataProviderV2V3.sol

```diff
@@ -72,7 +72,7 @@ contract UiPoolDataProviderV2V3 is IUiPoolDataProviderV3 {
   {
     IAaveOracle oracle = IAaveOracle(provider.getPriceOracle());
     ILendingPool lendingPool = ILendingPool(provider.getLendingPool());
-    address[] memory reserves = lendingPool.getReservesList(address asset);
+    address[] memory reserves = lendingPool.getReservesList(address pool);
```

Comment:
Where is pool from? where is it forst defined or imported?

Reply:
it is `address pool`so it is a param that's created inside that function.